src/component/application/__tests__/__snapshots__/application-edit-component-test.js.snap

```diff
@@ -324,7 +324,7 @@ exports[`renders correctly with permissions 1`] = `
                       className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                     >
                       <a
-                        href="/features/strategies/ToggleA"
+                        href="/projects/undefined/toggles/ToggleA/strategies/ToggleA"
```

Comment:
huh. "undefined"?

Reply:
Thanks. Feature toggle data in tests were missing "project" property.